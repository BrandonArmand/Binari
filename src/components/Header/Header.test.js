import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Header from './Header';

describe("Header", () => {
    it("renders correctly", () => {
        const tree = renderer
            .create(
              <Router>
                <Header brand="BrandName" route="/home" />
              </Router>
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
