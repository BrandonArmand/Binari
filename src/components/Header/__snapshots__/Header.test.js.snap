// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionFixed makeStyles-appBar-1 MuiAppBar-colorPrimary mui-fixed"
>
  <div
    className="MuiToolbar-root MuiToolbar-regular makeStyles-container-5 MuiToolbar-gutters"
  >
    <div
      className="makeStyles-flex-6"
    >
      <a
        className="makeStyles-link-2"
        href="/home"
        onClick={[Function]}
      >
        <button
          className="MuiButtonBase-root MuiButton-root makeStyles-title-7 MuiButton-text"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            BrandName
          </span>
          <span
            className="MuiTouchRipple-root"
          />
        </button>
      </a>
    </div>
    <div
      className="PrivateHiddenCss-smDown-24"
    />
  </div>
</header>
`;
